ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -45,9 +45,11 @@ func MergeManifests(individualMachineConfigs map[string]interface{}, doNotMerge
 	}
 
 	for roleName, machineConfigs := range mergableMachineConfigs {
-		var osImageURL string = ""
+		cconfig := &machineconfigv1.ControllerConfig{}
+		cconfig.Spec.OSImageURL = ""
+		cconfig.Spec.BaseOSContainerImage = ""
 		//It only uses OSImageURL provided by the CVO
-		merged, err := mcfgctrlcommon.MergeMachineConfigs(machineConfigs, osImageURL)
+		merged, err := mcfgctrlcommon.MergeMachineConfigs(machineConfigs, cconfig)
```

Comment:
now the CI failure is obvious. It will not be so obvious months down the line when we will check the git history. Please record the API changed in the commit message.

Reply:
Change the commit message with more info.